invoker.js

```diff
@@ -319,15 +319,6 @@ export function apply() {
     const source = event.target.closest("button[commandfor], button[command]");
     if (!source) return;
 
-    if (this.form && this.getAttribute("type") !== "button") {
```

Comment:
This should remain but should change to `source.form` I think.

Reply:
Done in [cacfc33](https://github.com/keithamus/invokers-polyfill/pull/56/commits/cacfc33184f3229c0930b49ec5536a6756ea014a).

The PR title and description have also been updated.